tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
-envlist = py36, py37, py38, lint
+envlist = py36, py37, py38, lint, mypy
 skip_missing_interpreters = True
 
 [gh-actions]
 python =
-  3.6: py36, lint
+  3.6: py36, lint, mypy
```

Comment:
We should consider dropping py36.

Reply:
It's 1.5 years until EOL for Python 3.6. I don't mind dropping it, but should we make a general policy? Should we say that we support the two latest Python versions?

Reply:
Yeah. Follow HA :)

Reply:
Ok, I'll drop 3.6 and document this in a separate PR.